.pre-commit-config.yaml

```diff
@@ -66,6 +66,14 @@ repos:
     stages: [manual]
     pass_filenames: false
 
+  - id: check-unused-components
+    name: Check for unused components
+    entry: bash -c '$(which bazel) run //rs/ic_os/manifest_tool -- --repo-root $(git rev-parse --show-toplevel) check-unused-components'
```

Comment:
I was forced to dynamically determine the correct Bazel path using which. Otherwise, pre-commit gave the error `/usr/bin/bash: /usr/local/bin/bazel: No such file or directory`

See pre-commit failure _without_ dynamically determining the Bazel path:
https://github.com/dfinity/ic/actions/runs/10426685939/job/28880092864